doc/user/layouts/partials/head.html

```diff
@@ -93,57 +93,19 @@
 </script>
 
 {{if hugo.IsProduction}} {{/* Google Analytics */}}
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-HYXTGZN2KT"></script>
 <script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'G-HYXTGZN2KT');
```

Comment:
Confirming that this is the same ID we use within our Google Tag Manager container
<img width="616" alt="image" src="https://github.com/user-attachments/assets/ff4a8152-e88a-43ef-b8fd-6946b76649b6">